packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -123,58 +123,6 @@ describe.skipIf(!process.env.RUNNING_IN_CI)(
       expect(decryptedMessageString).toEqual(messageString);
     }, 15000);
 
-    test('should encrypt and decrypt according to wallet allowlist condition', async () => {
```

Comment:
Running in integration test is very helpful. If not running it always, probably you may add one with `skip` that you run at least locally to confirm that everything is running as expected...

Reply:
Not sure I understand what you mean. Could you clarify? This test was removed because it was related to allowlist condition specifically, which was fully removed in this PR. It may be added back in https://github.com/nucypher/sprints/issues/206.

Reply:
I mean to add (or modify) an integration test that tests ContextVariableCondition.